features/product/viewing_products/viewing_products_price.feature

```diff
@@ -0,0 +1,14 @@
+@viewing_products
+Feature: Viewing a product price
```

Comment:
I'm not sure. It is almost the same, but first action is on taxon list and the second one is on product page.